data-prepper-logstash-configuration/src/main/java/com/amazon/dataprepper/logstash/model/LogstashAttribute.java

```diff
@@ -0,0 +1,22 @@
+package com.amazon.dataprepper.logstash.model;
+
+public class LogstashAttribute {
```

Comment:
We should be adding Javadocs to public classes. These should include a description of what it provides and also an `@since VERSION` attribute. In this case, `@since 1.2`.

Here is a simple example:
https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-core/src/main/java/com/amazon/dataprepper/plugin/DefaultPluginFactory.java#L14-L18